### ISSUE TITLE
On 3.3.20.1 no fields are changing.

### ISSUE THREAD

**Issue by ryanpcmcquen** [venveo/craft-bulkedit#16]
I'm changing plain text fields on a few articles, and I see the request being made:
```
/admin/entries/articles?elementIds=376122%2C923538%2C923513%2C738502&elementType=craft%5Celements%5CEntry&siteId=1&fieldMeta%5B0%5D%5Bid%5D=130&fieldMeta%5B0%5D%5Bstrategy%5D=replace&fields%5Bdescription%5D=FOOBARBAZ
```

But nothing ever actually changes. I don't see anything added to the queue either. Am I missing something easy?

**Comment by Mosnar** [comment]
Haven't seen that before! What version are you using?

**Comment by ryanpcmcquen** [comment]
3.3.20.1

**Comment by Mosnar** [comment]
I just did a test on a fresh install of Craft 3.3.20.1 and Bulk Edit 2.0 and didn't have any trouble performing a bulk replace on a text field. 

Here's a quick video of what success should look like: https://venveo.d.pr/h0p8zo

To help debug, could you first clear your CP Resources cache and then check for any console errors you get when saving? That request you posted definitely doesn't look correct - there should be 3 XHR requests during the process total:
1) When you open the bulk edit dialog: `index.php?p=admin/actions/venveo-bulk-edit/bulk-edit/get-fields`
2) When you select the fields to edit and click next: `index.php?p=admin/actions/venveo-bulk-edit/bulk-edit/get-edit-screen`
3) And finally when you click Save: `index.php?p=admin/actions/venveo-bulk-edit/bulk-edit/save-context`

**Comment by ryanpcmcquen** [comment]
Weirdness, that third request never happens for me, here's a gif:
![bulk_edit_sample](https://user-images.githubusercontent.com/772937/74473485-85f90700-4e58-11ea-9235-4862765b7c69.gif)

**Comment by ryanpcmcquen** [comment]
The only thing I can find in the logs related to Venveo after those two requests is this:
```
2020-02-13 12:00:24 [-][140926][-][profile end][Twig template] render template: _components/elementactions/SetStatus/trigger
2020-02-13 12:00:24 [-][140926][-][info][yii\db\Command::execute] INSERT INTO `resourcepaths` (`path`, `hash`) VALUES ('@venveo/bulkedit/assetbundles/bulkeditelementaction/dist', 'd972c0fc') ON DUPLICATE KEY UPDATE `path`='@venveo/bulkedit/assetbundles/bulkeditelementaction/dist'
2020-02-13 12:00:24 [-][140926][-][profile begin][yii\db\Command::execute] INSERT INTO `resourcepaths` (`path`, `hash`) VALUES ('@venveo/bulkedit/assetbundles/bulkeditelementaction/dist', 'd972c0fc') ON DUPLICATE KEY UPDATE `path`='@venveo/bulkedit/assetbundles/bulkeditelementaction/dist'
2020-02-13 12:00:24 [-][140926][-][profile end][yii\db\Command::execute] INSERT INTO `resourcepaths` (`path`, `hash`) VALUES ('@venveo/bulkedit/assetbundles/bulkeditelementaction/dist', 'd972c0fc') ON DUPLICATE KEY UPDATE `path`='@venveo/bulkedit/assetbundles/bulkeditelementaction/dist'
```

Would you mind if I emailed you our plugin list? Maybe we have something that conflicts there?

**Comment by Mosnar** [comment]
I think I've got it figured out actually! Was able to reproduce in Firefox and found that the form submit event wasn't getting caught. I just released 2.0.1, could you give that a try and let me know if that solves the problem?

**Comment by Mosnar** [comment]
Assuming by  🎉  the issue is resolved. Feel free to comment and let me know if that's not the case!

**Comment by Mosnar** [comment]
Fixed in a26aed2a5036e648a19b2a346c3724a08afad7ff

**Comment by ryanpcmcquen** [comment]
@Mosnar, it works! Thank you so much!
